### PR TITLE
Feature/col 2126 package update

### PR DESCRIPTION
**Pull request by martonx**
As IE removed as supported browser, I removed some unnecessary polyfills besides npm package update.
And karma config is updated for our current browser requirements.

### CODE REVIEW

File: test/karma.conf.js [hunk 1]
```diff
@@ -18,7 +18,10 @@ module.exports = function (config) {
         },
         browserify: {
             debug: true,
-            transform: [['babelify', { 'presets': ['es2015'] }]]
+            transform: [['babelify', {
+                "presets": ["@babel/preset-env"],
```
Comment (lvass74) [inline test/karma.conf.js]: if preset-env without specifying any targeted browser is equivalent to es2015 language level, than I'm fine with this change.

Comment (achueshkov) [inline test/karma.conf.js]: good point! they are not equal. the changes should not go live until this is not fixed

Comment (martonx) [reply test/karma.conf.js]: transform synced with our other components babel transform setup

Comment (martonx) [reply test/karma.conf.js]: transform synced with our other components babel transform setup

File: test/karma.conf.js [hunk 2]
```diff
@@ -18,7 +18,29 @@ module.exports = function (config) {
         },
         browserify: {
             debug: true,
-            transform: [['babelify', { 'presets': ['es2015'] }]]
+            transform: [['babelify', {
+                "presets": [
+                    [
+                        "@babel/preset-env",
```
Comment (achueshkov) [inline test/karma.conf.js]: those presets for webrtc, they are not equal to es2015. also in karma config we can see lines about ie11, what is contradictionarary here

Comment (martonx) [reply test/karma.conf.js]: those presets not just for webrtc, but for our current browser compatibility requirements: https://jira.realeyesit.com/confluence/display/DC/Supported+Platforms

I synced karma conf with this preset, IE removed, bs browsers updated for this requirement.

**Pull request closed without merging**